adapter/stdLogger/config/config.proto

```diff
@@ -0,0 +1,79 @@
+// Copyright 2016 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package stdLogger.config;
+
+option go_package = "config";
+
+message Params {
+
+    // Stream is used to select between different logs output sinks.
+    // STDERR is the default Stream.
+    enum Stream {
+
+        // STDERR refers to os.Stderr.
+        STDERR = 0;
+
+        // STDOUT refers to os.Stdout.
+        STDOUT = 1;
+    }
+
+    // Selects which standard stream to write to for log entries.
+    Stream log_stream = 1;
+
+    // Provides a way to identify a collection of related logs within the
+    // single stream. If no log_name is provided, a default value of
+    // "istio_log" will be used.
+    string log_name = 2;
+
+    // PayloadFormat details the currently supported logging payload formats.
+    // TEXT is the default payload format.
+    enum PayloadFormat {
+
+        // Indicates a payload format of raw text.
+        TEXT = 0;
+
+        // Indicates a structured payload (JSON object).
+        STRUCT = 1;
+    }
+
+    // The name of the attribute to treat as the payload of the logs message.
+    // If payload_attribute is not specified, there will be no payload specified
+    // in the log entry written. All provided attributes will be written out
+    // as logging metadata.
+    string payload_attribute = 3;
+
+    // Format of the value of the payload attribute.
+    PayloadFormat payload_format = 4;
+
+    // The name of the attribute that will be used to derive log severity.
+    // If this attribute is not found in the set of attributes passed to this
+    // implementation, a default severity of INFO will be used.
+    string severity_attribute = 5;
```

Comment:
Should be this be a generic thing in the LogEntryDescriptor? Or should it be a generic config for all logging aspects?